backup.yml

```diff
@@ -0,0 +1,19 @@
+# file: backup.yml
+
+- name: Backup OpenNMS
+  hosts: core
+  user: root
```

Comment:
Depends if the Ansible user has enough permissions. For the copy jobs of the files it could work with `ubuntu`. But maybe not for the PSQL dump.
Right now it is set to `root` since I also don't have an `ubuntu` user.